hms-app/src/App.tsx

```diff
@@ -26,6 +26,7 @@ function App() {
       <Route path='/' element={<Layout />}>
         <Route index element={<Home />} />
         <Route path='ideas' element={<AllIdeas />} />
+        <Route path='ideas/:slug' element={<AllIdeas />} />
```

Comment:
I think this is the more common path.  I wonder though if it should be like `idea/slug/:slug` or just `slug/:slug`.  That way we could still have `idea/:idea_id` 🤔 

```suggestion
        <Route path='idea/:slug' element={<AllIdeas />} />
```

Reply:
I also call this "more common" based mostly on how github does `...github.com.../pulls` and `...github.com.../pull/1252`

Reply:
we shouldnt use the word slug in the url as it could confuse people but maybe call it hackathons/:slug to make it easier to understand 

Maybe if we decide to have own pages for the idea details we could use something like idea/:ideaId